ipalib/config.py

```diff
@@ -497,6 +498,10 @@ def _bootstrap(self, **overrides):
         if 'plugins_on_demand' not in self:
             self.plugins_on_demand = (self.context == 'cli')
 
+        # Set fips_mode:
+        if 'fips_mode' not in self:
+            self.fips_mode = tasks.is_fips_enabled()
```

Comment:
Do we want the user to be able to set env.fips_mode during api bootstrapping? I do not think so. This should be set amongst other run-time variables.